examples/helm/ha_cluster_with_proxy_and_password.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#
+# This values file overrides the default kube-starrocks chart to:
+# - add a default password for the root user
+# - deploy a highly-available StarRocks cluster with 3 FE pods and 3 BE pods
+# - deploy a LoadBalancer for the FE service
+# - deploy a LoadBalancer for the FE Proxy service
+# Note: This is not a production-level cluster, the CPU, Memory, and Disk resources
+# are set very low to allow this to be deployed in a Kubernetes cluster with limited
+# resources. For production sizing information please see:
+# https://docs.starrocks.io/docs/deployment/plan_cluster/
+#
+# To use this values file:
+# - [Add the Helm repo](../../doc/add_helm_repo_howto.md)
+# - Set a passord for the StarRocks database root user:
+#   kubectl create secret generic starrocks-root-pass --from-literal=password='g()()dpa$$word'
+# - Download this file:
+#   curl -O https://raw.githubusercontent.com/StarRocks/starrocks-kubernetes-operator/main/examples/examples/helm/ha_cluster_with_proxy_and_password.yaml
+# - Deploy:
+#   helm install -f ha_cluster_with_proxy_and_password.yaml starrocks starrocks-community/kube-starrocks
+#
+# Once the operator and cluster are ready you can access the system with a MySQL
+# client at the IP Address of the kube-starrocks-fe-service on port 9030.
+#
+# For example:
+# mysql -h < kube-starrocks-fe-service IP > -P9030 -uroot -p
+# 
+# Note: You will be prompted for the password you set in the secret earlier
+#   
+starrocks:
+    initPassword:
+        enabled: true
+            # Set a password secret, for example:
+            # kubectl create secret generic starrocks-root-pass --from-literal=password='g()()dpa$$word'
+        passwordSecret: starrocks-root-pass
+
+    starrocksFESpec:
+        replicas: 3
+        service:
+            type: LoadBalancer
+        resources:
+            requests:
+                cpu: 1
+                memory: 1Gi
+
+    starrocksBeSpec:
+        replicas: 3
+        resources:
+            requests:
+                cpu: 1
+                memory: 2Gi
+        storageSpec:
```

Comment:
should add name field. https://github.com/StarRocks/starrocks-kubernetes-operator/blob/036fdd502ee78b091b3ff18a0dcbbd083baaedc7/helm-charts/charts/kube-starrocks/values.yaml#L384 for the suggested naming rule